docs/client-libraries-cpp.md

```diff
@@ -21,7 +21,7 @@ You can use a Pulsar C++ client to create Pulsar [producers](concepts-clients.md
 
 ## Reference doc
 
-- [API docs](@pulsar:apidoc:cpp@)
+- [API docs](https://pulsar.apache.org/api/cpp/)
```

Comment:
This can be a handy fix. Since now C++ client has it own release cycle, I can do a global review of those links and fix https://github.com/apache/pulsar/issues/19121.

Let me prepare another patch :)

Reply:
https://github.com/apache/pulsar-site/pull/452

Reply:
```suggestion
- [API docs](@pulsar:apidoc:cpp@)
```

Reply:
Thank you!